deploy/complete/helm-chart/provision/templates/cluster-service-broker.yaml

```diff
@@ -2,7 +2,7 @@
 apiVersion: servicecatalog.k8s.io/v1beta1
 kind: ClusterServiceBroker
 metadata:
-  name: {{.Values.global.osb.name}}
+  name: oci-cluster-service-broker
```

Comment:
differentiate the oci-broker from the cluster service broker resource

Reply:
<img width="428" alt="image" src="https://user-images.githubusercontent.com/551007/73952687-e3271280-48c4-11ea-9332-3e3e35053dca.png">